canonical-web-and-design/snapcraft.io
# Convert brand store section nav to TypeScript

## Done
Converted brand store `SectionNav` component to Typescript

## QA
- Go to https://snapcraft-io-3801.demos.haus/admin
- Check that the section nav works

## Issue
Fixes #3774
